src/stories/components/HorizontalTitle/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// HorizontalTitle
+
+.horizontal-title {
+    font-family: 'Work Sans';
+    padding-top: 0;
+    margin-top: 0;
+
+    &--medium {
+        .horizontal-title__title {
+            font-size: 20px;
```

Comment:
Font sizes are already declared in a global variables. Replace with `$font-size-lg`.